doc/appendix-de-kassensichv/reference-tables/reference-tables.md

```diff
@@ -279,3 +279,478 @@ For definitions regarding national laws, please refer to the appropriate appendi
+
+##### File: TSE_Transaktionen (transactions_tse.csv)
+
+| **Fieldname**            | **Description**          | **Format**          | **ft.input** |
+|----------------------|--------------------------|---------------------|---------------------|
+| `Z_KASSE_ID` | ID of the (closing) cashpoint | String (50) | Automatically created and filled by ft (`ftCashBoxIdentification`) |
+| `Z_ERSTELLUNG` | Date of the cashpoint closing | ISO 8601 und RFC3339 date | Automatically filled by ft from `cbReceiptMoment` of the daily closing receipt |
+| `Z_NR` | Nr. of the cashpoint closing | Integer | Automatically created and filled by ft |
+| `BON_ID` | Action-ID | String (40) | Automatically created and filled by ft (`ftReceiptIdentification`) |
+| `TSE_ID` | ID of the TSE used for the transaction | Integer | Automatically filled by ft |
+| `TSE_TANR` | Transaction number of the transaction | Integer | Automatically filled by ft|
+| `TSE_TA_START` | Log time of the StartTransaction operation | String (30) | Automatically filled by ft |
+| `TSE_TA_ENDE` | Log time of the FinishTransaction operation | String (30) | Automatically filled by ft |
+| `TSE_TA_VORGANGSART` | processType der FinishTransaktions operation | String (30) | Automatically filled by ft |
+| `TSE_TA_SIGZ` | Signature counter of the FinishTransaction operation | Long | Automatically filled by ft |
+| `TSE_TA_SIG` | Signature of the FinishTransaction operation | String (512) | Automatically filled by ft |
+| `TSE_TA_FEHLER` | Where appropriate, indications of TSE errors | String (200) | Automatically filled by ft|
+| `TSE_VORGANGSDATEN` | Data of the operation (optional) | String | Automatically filled by ft |
+
+#### Master data module (DE: Stammdatenmodul)
+
+To avoid redundancies, the master data is only saved once for each cashpoint closing. If changes are made to the master data listed below, a closing must be created automatically beforehand.
+
+The **master data** for the day that is closed by the daily closing receipt (or monthly, yaearly because they include the daily closing) **must be included into the closing receipt** (daily/monthly/yearly). To include the data please send the content of `ftReceiptCaseData` in JSON format and add the key value pair `dailyClosingMasterData`. The value should also be JSON and formatted as follows:
+```json
+"dailyClosingMasterData" : "{ 
+
+    "CompanyName" : "...",
+    "CompanyStreet" : "...",
+    "CompanyZip" : "...",
+    "CompanyCity" : "...",
+    "CompanyCountry" : "...",
+    "CompanyTaxNr" : "...",
+    "CompanyVATId" : "...",
+    
+    "LocationName" : "...",
+    "LocationStreet" : "...",
+    "LocationZip" : "...",
+    "LocationCity" : "...",
+    "LocationCountry" : "...",
+    "LocationVATId" : "...",
+    
+    "CashRegisterBrand" : "...",
+    "CashRegisterModel" : "...",
+    "CashRegisterSWBrand" : "...",
+    "CashRegisterSWBVersion" : "...",
```

Comment:
Typo: Change "CashRegisterSWBVersion" to "CashRegisterSWVersion"

Reply:
thank you, this has been removed for now. it will be part of the master data processing